cmd/vela-terraform/validate.go

```diff
@@ -50,23 +50,20 @@ func (v *Validation) Command(dir string) *exec.Cmd {
 
 	// check if Vars is provided
 	if len(v.Vars) > 0 {
-		var vars string
 		for _, v := range v.Vars {
-			vars += fmt.Sprintf("-var=%s ", v)
+			flag := fmt.Sprintf(`-var=%s`, v)
+			// add flag for Vars from provided command
+			flags = append(flags, flag)
```

Comment:
Do you think we could condense this into one line?

```suggestion
			// add flag for Vars from provided command
			flags = append(flags, fmt.Sprintf(`-var=%s`, v))
```